### PR TITLE
pytorch porting of the (tensorflow) pretrained network

### PR DESCRIPTION
**Pull request by ncianeo**
Hello,

I successfully ported your tensorflow pretrained network to pure pytorch model.
Check "pytorch_test" folder

JOB description:
- convert tf checkpoint files to a python dictionary like, {"generator.Conv.weight": weight_params(np.ndarray), ...}
- write a pytorch model with keeping its modules having the same name with the keys of the dictionary (ex. self.Conv = nn.Conv2d and so on ...)
- test model loading function & test on images...

### CODE REVIEW

**Comment by ncianeo** [comment]
There were several jobs to be done:
1. tf uses asymmetric padding (k_size//2-1, k_size) when using 'SAME' padding, but pytorch uses symmetric padding (k_size//2, k_size//2) --> I wrote tf_same_padding function in model.py
2. bilinear interpolation in tf behaves differently from other computer vision libraries(opencv, pytorch vision functions, etc)... --> I wrote tensorflow-like bilinear upsampling(with scale factor=2) in the function tf_upsample_bilinear in model.py

**Comment by aba-ai-learning** [comment]
Hello, Could you please tell me how do you transfer the model.index  model.ckpt to pytorch pth ?

**Comment by aba-ai-learning** [comment]
Hello, Could you please tell me how do to convert tf checkpoint files to a python dictionary
